package.json

```diff
@@ -97,7 +98,7 @@
     "@dnd-kit/core": "^3.1.1",
     "@dnd-kit/sortable": "^4.0.0",
     "@dnd-kit/utilities": "^2.0.0",
-    "@elastic/apm-rum": "^5.9.1",
+    "@elastic/apm-rum": "^5.10.0",
```

Comment:
This update looks safe, and the new version already exposes our needed type: `Transaction`
(https://www.elastic.co/guide/en/apm/agent/rum-js/current/release-notes-5.x.html#release-notes-5.10.0)